pepper-apis/config/redisson-jcache.yaml.ctmpl

```diff
@@ -15,7 +15,7 @@ singleServerConfig:
   subscriptionConnectionMinimumIdleSize: 1
   subscriptionConnectionPoolSize: 50
   connectionMinimumIdleSize: 24
-  connectionPoolSize: 86
+  connectionPoolSize: 64
   database: 0
```

Comment:
We have two separate Redis clients being created. Let's back off a bit on the connections. There has been no evidence that this was the problem in the first place.